packaging/bundle-validation/ci_run.sh

```diff
@@ -162,6 +162,8 @@ else
     HUDI_FLINK_BUNDLE_NAME=hudi-flink1.16-bundle
   elif [[ ${FLINK_PROFILE} == 'flink1.17' ]]; then
     HUDI_FLINK_BUNDLE_NAME=hudi-flink1.17-bundle
+  elif [[ ${FLINK_PROFILE} == 'flink1.18' ]]; then
+    HUDI_FLINK_BUNDLE_NAME=hudi-flink1.18-bundle
```

Comment:
The `IMAGE_TAG` should be updated as flink 1.18 after we uploaded the docker image. cc @codope for the help ~